### DISCUSSION TITLE
Committen sonra github uzantısında classroom sekmesi çıkmıyor

### DISCUSSION THREAD

**Discussion by onurgnll** [19/classroom#24] [Anonim] [answered]
![Ekran Alıntısı](https://user-images.githubusercontent.com/92329981/158044912-e1810726-6f16-4a80-a7be-f9d080ee8b2d.PNG)
Ödevi commitledim fakat puanımı görebileceğim gibi bir sekme açılmıyor. Sorun oluşturur mu?

**Comment by onurgnll** [comment, marked as answer]
![Ekran Alıntısı2](https://user-images.githubusercontent.com/92329981/158045015-31b69632-54ff-4630-8376-8df0bc56ae40.PNG)
Ek olarak git hub ödev sayfasında commitlendiği görülebiliyor

**Reply by gulcanyildiz** [reply]
Ödevin teslim edilmiş. Sorun yok.
